templates/submission/status-testcases.html

```diff
@@ -33,7 +53,7 @@ <h3>{{ _('Execution Results') }}</h3>
                 <br>
                 <div class="batch-cases">
             {% endif %}
-        <table>{% for case in batch.cases %}
+        <table id="submissions-status-table">{% for case in batch.cases %}
```

Comment:
`id` is meant to be globally unique, whereas a submission may have multiple batches.